### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by rebelEpik**
Fixed SettingsManager gui to display properly for Market Settings
Added setting for sound on next market item
Added notification setting for eventual toast notifications
Cleaned up minor errors in code
Fixed UndercutBy binding on SettingsManager to use the correct backing property

### CODE REVIEW

File: REvernus/Settings/AppSettings.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@ public class AppSettings : ITrackingAware<AppSettings>
         public MarginToolSettings MarginToolSettings { get; set; } = new MarginToolSettings();
         public MarketOrdersTabSettings MarketOrdersTabSettings { get; set; } = new MarketOrdersTabSettings();
         public MarketSettings MarketSettings { get; set; } = new MarketSettings();
+        public NotificationSettings NotificationSettings { get; set; } = new NotificationSettings();
```
Comment (Meigs2) [inline REvernus/Settings/AppSettings.cs]: Notifications should not be part of this branch, code relating to notifications should be removed entirely.

File: REvernus/Settings/NotificationSettings.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace REvernus.Settings
+{
+    public class NotificationSettings
```
Comment (Meigs2) [inline REvernus/Settings/NotificationSettings.cs]: Again, this class should be deleted

File: REvernus/ViewModels/MarketOrdersViewerViewModel.cs [hunk 3]
```diff
@@ -398,7 +407,7 @@ private async Task ImportOrders()
                 }
 
                 await Task.WhenAll(taskList);
-
+                GenerateOverbidNotifications(BuyOrdersCollection, SellOrdersCollection);
```
Comment (Meigs2) [inline REvernus/ViewModels/MarketOrdersViewerViewModel.cs]: Function should be removed.

File: REvernus/Views/MainWindowView.xaml [hunk 4]
```diff
@@ -49,7 +49,18 @@
             <Grid>
                 <TabControl x:Name="MainTabControl">
                     <TabItem Header="Market Orders">
-                        <views:MarketOrdersViewerView/>
+                        <TabControl>
+                            <TabItem Header="Buy/Sell">
+                                <views:MarketOrdersViewerView/>
+                            </TabItem>
+                            <TabItem Header="Buy">
```
Comment (Meigs2) [inline REvernus/Views/MainWindowView.xaml]: What is the purpose of these TabItems? They seem like they should be in MarketOrderViewerView.xaml, not MainWindowView.xaml

Comment (rebelEpik) [reply REvernus/Views/MainWindowView.xaml]: Removed

File: REvernus/Settings/NotificationSettings.cs (hunk 2 above)
Comment (rebelEpik) [reply REvernus/Settings/NotificationSettings.cs]: Removed

File: REvernus/Settings/AppSettings.cs (hunk 1 above)
Comment (rebelEpik) [reply REvernus/Settings/AppSettings.cs]: Removed

**Comment by Meigs2** [comment]
Looks good!

**Pull request merged**